packages/components/src/components/icon/appstore/ic-appstore-advcash-dark.svg

```diff
@@ -0,0 +1 @@
6 0 0 1 .215.33l-.244.086c-.043-.2-.172-.33-.345-.33-.215 0-.359.172-.359.445 0 .259.144.431.36.431.186 0 .301-.115.344-.33l.244.1z" fill="#fff"/><path fill-rule="evenodd" clip-rule="evenodd" d="M31.085 51.642c-.345 0-.646-.115-.89-.36a1.181 1.181 0 0 1-.36-.875c0-.345.115-.647.36-.876a1.19 1.19 0 0 1 .89-.36c.344 0 .646.115.876.36.244.244.359.53.359.876 0 .344-.115.646-.36.876-.229.23-.53.359-.875.359zm.746-.489c.202-.2.302-.46.302-.76 0-.302-.1-.547-.302-.748a1.007 1.007 0 0 0-.746-.301 1.03 1.03 0 0 0-1.049 1.048 1.03 1.03 0 0 0 1.048 1.063c.288 0 .546-.1.747-.302zM35.48 51.555h3.288l1.45-4.136h7.066l1.565 4.136h3.59l-6.807-17.707h-3.59L35.48 51.555zM46.22 44.59h-4.983l2.427-6.807 2.556 6.807zM61.027 51.555v-3.418c-.445 2.283-1.68 3.662-4.02 3.662-2.413 0-4.812-1.939-4.812-6.204 0-4.136 2.413-6.075 4.811-6.075 2.341 0 3.562 1.35 4.021 3.62v-9.292h3.074v17.707h-3.074zm-2.656-1.924c1.536 0 2.57-1.422 2.57-4.065 0-2.556-1.034-3.963-2.57-3.963-1.537 0-2.758 1.422-2.758 3.963-.014 2.657 1.22 4.065 2.758 4.065z" fill="#fff"/><path d="M70.333 51.555h2.126L77.7 39.75h-2.972l-3.074 7.698-3.015-7.698h-3.375l5.07 11.805zM83.631 39.506c2.456 0 4.74 1.566 5.314 4.022l-2.815 1.034c-.402-2.169-1.393-3.074-2.685-3.074-1.58 0-2.614 1.465-2.614 4.136 0 2.858 1.25 4.136 2.642 4.136 1.436 0 2.198-1.106 2.671-3.303l2.815 1.034c-.445 2.614-2.815 4.294-5.313 4.294-3.418 0-6.233-2.527-6.233-6.16-.014-3.591 2.8-6.119 6.218-6.119zM107.815 44.274c-1.895-.402-2.484-.732-2.484-1.364 0-.66.56-1.106 1.694-1.149 1.034 0 1.91.43 2.499 1.565l2.097-1.062c-.876-2.054-2.743-2.786-4.725-2.786-2.384 0-4.323 1.263-4.323 3.619 0 2.197 1.293 3.245 4.036 3.805 1.895.402 2.484.804 2.484 1.494 0 .632-.502 1.034-1.752 1.034-1.321 0-2.412-.489-3.001-1.623l-2.097 1.177c.876 1.867 2.571 2.787 5.026 2.787 3.289 0 4.582-1.767 4.582-3.82 0-1.953-1.02-3.045-4.036-3.677zM116.446 51.555h-3.073V33.848h3.073v9.78c.445-2.614 1.68-4.108 4.021-4.108 1.838 0 3.547 1.25 3.547 3.706v8.329h-3.073v-7.18c0-.546-.172-1.035-.474-1.537-.301-.503-.804-.733-1.493-.733-1.896 0-2.528 1.91-2.528 2.83v6.62z" fill="#fff"/><path fill-rule="evenodd" clip-rule="evenodd" d="m90.223 43.183 2.815 1.034c.244-.862.517-1.465.833-1.795.33-.374.933-.56 1.81-.56.732 0 1.249.143 1.565.445.316.301.473.761.473 1.393v.502c-1.091.403-2.096.69-3.03.877-.933.186-1.737.402-2.412.631-.675.23-1.207.575-1.594 1.006-.388.43-.575 1.091-.575 1.967 0 .977.316 1.738.934 2.298.617.56 1.45.833 2.47.833.43 0 .876-.072 1.35-.201.008-.008.02-.012.035-.016a3.766 3.766 0 0 0 1.832-1.234l.99-1.206v.546l-.014 1.867h3.088v-8.387c0-1.307-.388-2.24-1.164-2.815-.775-.56-2.053-.847-3.834-.847-.675 0-1.321.072-1.939.215a5.455 5.455 0 0 0-1.651.66 4.54 4.54 0 0 0-1.25 1.15 4.268 4.268 0 0 0-.732 1.637zm3.648 5.988c-.273-.273-.402-.617-.402-1.062 0-.675.287-1.221.861-1.638.436-.304 1.193-.527 2.266-.841.342-.101.717-.211 1.123-.336v1.896A3.515 3.515 0 0 1 96.513 49c-.244.186-.517.33-.79.43-.273.1-.517.144-.718.144-.488 0-.861-.13-1.134-.403z" fill="#fff"/></svg>
```

Comment:
@Bahar-FS All these icons are already available in `appstore/src/assets/svgs/wallet`? Can we use it from there?